tools/docker/docker-compose-mac.yml

```diff
@@ -168,5 +168,16 @@ services:
       retries: 3
       start_period: 5s
 
+  squid:
+    profiles:
+      - proxy
+    image: 'quay.io/openshifttest/squid-proxy:1.2.0'
+    hostname: squid
+    volumes:
+      - ./squid/squid.conf:/etc/squid/squid.conf
```

Comment:
@bzwei Have you tested this with podman and rootless containers?

Reply:
yes, it works in my podman provision